docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc

```diff
@@ -0,0 +1,24 @@
+= Identifying Policy in-use
+
+In Konflux, there are two places where Enterprise Contract Policies are used, integration tests and
+release pipeline.
+
+
+== Integration Tests
+
+For integration tests, the Policy is defined in an IntegrationTestScenario resource. These are
+stored in the same namespace as your Application. They are created automatically when a new
+Application is created via the Web UI.
+
+IntegrationTestScenarios have multiple purposes. Running EC validation is just one of them. As such,
+you may have multiple of these resources in your namespace. The one that runs EC points to
+`pipelines/enterprise-contract.yaml` Tekton Pipeline in the
+`https://github.com/konflux-ci/build-definitions` repository.
+
+Update, or set, the value of the `POLICY_CONFIGURATION` parameter to your new Policy.
```

Comment:
Consider if we should we talk about the so many ways to do it, e.g. go-getter url formats, the way ec tries to find a policy.json or policy.yaml, the option to reference an ECP record in the cluster.

Reply:
The details probably belong in the upstream docs. I think in the context of Konflux, hopefully this should be somewhat obvious as users are already tasked in updating the parameter during the onboarding process.

Reply:
Is there any place in the EC doc where you can link to for more information?

Reply:
I couldn't find it. Filed https://github.com/enterprise-contract/ec-cli/issues/1834.